pkg/bootstrap/platform/aws.go

```diff
@@ -15,83 +15,66 @@
 package platform
 
 import (
-	"errors"
-	"io/fs"
-	"os"
-	"strings"
-
-	"github.com/aws/aws-sdk-go/aws"
```

Comment:
Can the dependency on github.com/aws/aws-sdk-go be removed from go.mod now that is no longer used in aws.go and aws_test.go?

Reply:
Yes. It is removed now.